router/cmd/boot/boot.go

```diff
@@ -62,6 +65,9 @@ func main() {
 
 	log.Info("Starting Nginx...")
 
+	go tailFile(nginxAccessLog)
+	go tailFile(nginxErrorLog)
+
 	nginxChan := make(chan bool)
 	go launchNginx(nginxChan)
 	<-nginxChan
```

Comment:
yes, but that code didn't work as intended since it never waited for the goroutine to finish. I'll fix that up as well.